indexer-app/src/cmr/indexer/services/index_set_service.clj

```diff
@@ -18,7 +18,8 @@
 
 (def searchable-concept-types
   "Defines the concept types that are indexed in elasticsearch and thus searchable."
-  [:collection :granule :deleted-granule :tag :variable :service :autocomplete :subscription])
+  [:collection :granule :deleted-granule :tag :variable :service :autocomplete
```

Comment:
Since we're having to span multiple lines we might want to break them all apart by newline and alphabetize now.

Reply:
Done.